### PR TITLE
screenshot change

### PR DESCRIPTION
**Pull request by Ilithy**
Proposed screenshot change.
* I added the translation engine selection screen (image option-1.png and option-2.png).
* I did not add the history screen _(we can clearly see the option on the captures mentioned above)_
* I added the files _(trying to follow the naming style)_ in the folder `fastlane/metadata/android/en-US/images/phoneScreenshots/`
* I modified the Readme to display the new captures _(in addition to the "old ones")_

If you don't like it, feel free to refuse it.
Thanks

### CODE REVIEW

**Comment by Bnyro** [comment]
The new screenshots look good to me, thanks <3
It seems that you've been naming the screenshots `option-1` and `option-2` while referencing them with `options-1` and `options-2` in the README, therefore they don't load properly in the Readme yet :)

File: README.md [hunk 1]
```diff
@@ -29,7 +29,8 @@
 
 <div style="display: flex;">
   <img src="fastlane/metadata/android/en-US/images/phoneScreenshots/translate.png" width=30%>
-  <img src="fastlane/metadata/android/en-US/images/phoneScreenshots/options.png" width=30%>
+  <img src="fastlane/metadata/android/en-US/images/phoneScreenshots/options-1.png" width=30%>
```
Comment (Bnyro) [inline README.md:32]:
```suggestion
  <img src="fastlane/metadata/android/en-US/images/phoneScreenshots/option-1.png" width=30%>
```

**Comment by Ilithy** [comment]
> The new screenshots look good to me, thanks <3 It seems that you've been naming the screenshots `option-1` and `option-2` while referencing them with `options-1` and `options-2` in the README, therefore they don't load properly in the Readme yet :)

--'
Indeed, a mistake on my part, my apologies and thank you for the correction

**Comment by Bnyro** [comment]
> Indeed, a mistake on my part, my apologies and thank you for the correction
> 
No worries :)

> _PS: On the "about" screenshot, I removed the version number, to avoid having to modify it unnecessarily at each new update of the application_
>
That's indeed a good idea, thank you for your  efforts <3

**Pull request merged**